### DISCUSSION TITLE
Public Key

### DISCUSSION THREAD

**Discussion by ian-tiggeman** [ian-tiggeman/T24S2T1EXTG08#6] [General]
Hi Cara,
Have you have a chance to send me your public key yet?
You will need to do this before you can push back to the GitHub project (I think).
Send via email, so it more secure than on this discussions page, and let me know once done.
Thanks. :)

**Comment by DanyaoWang-001167576** [comment]
Hi Ian,
Thanks for reminding me about the public key. I’ve just sent it through. Please check your email :)
Cheers

**Reply by ian-tiggeman** [reply]
Thanks for sending the key.
I'm not sure what I am meant to do with it now.  The assignment say "add into the  key ring of the GitHub repository", but I can't find anything repository related for SSH keys.  If I try adding it to my GitHub account it say "key is already in use".
I really hate this subject's instructions.
